src/Catalyst.Core.Lib/CoreLibProvider.cs

```diff
@@ -152,7 +153,8 @@ protected override void Load(ContainerBuilder builder)
             builder.RegisterType<UploadFileTransferFactory>().As<IUploadFileTransferFactory>().SingleInstance();
 
             // Transaction validators
-            builder.RegisterType<TransactionValidator>().As<ITransactionValidator>().SingleInstance();
+            // builder.RegisterType<TransactionValidator>().As<ITransactionValidator>().SingleInstance();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4585091)